tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -129,12 +130,10 @@ def __init__(self, metadata: TestSuiteMetadata) -> None:
         # The context is initialized by _set_context() via the call to execute()
         self.__context: AgentTestSuite._Context = None
 
-    def _set_context(self, node: Node, variables: Dict[str, Any], lisa_log_path: str, log: Logger):
+    def _initialize(self, node: Node, variables: Dict[str, Any], lisa_working_path: str, lisa_log_path: str, lisa_log: Logger):
         connection_info = node.connection_info
         node_context = get_node_context(node)
         runbook = node.capability.get_extended_runbook(AzureNodeSchema, AZURE)
-        # Remove the resource group and node suffix, e.g. "e1-n0" in "lisa-20230110-162242-963-e1-n0"
-        runbook_name = re.sub(r"-\w+-\w+$", "", runbook.name)
```

Comment:
this was use to create a unique directory name for the runbook; no longer needed